runtime/Cargo.toml

```diff
@@ -40,6 +62,26 @@ default = ["setup-tracing"]
 api-client-tracing = ["shuttle-api-client/tracing"]
 
 setup-tracing = [
-    "tracing-subscriber/default",
+    "tracing-subscriber/ansi",
     "tracing-subscriber/env-filter",
+    "tracing-subscriber/fmt",
+    "tracing-subscriber/smallvec",
+    "tracing-subscriber/std",
+    "tracing-subscriber/tracing-log",
+]
+setup-telemetry = [
```

Comment:
I had an idea: This can be called `telemetry`, and we can rename `setup-tracing` to `tracing-subscriber`.

Reply:
Hmm yeah, and `telemetry` sounds like you'll be opting in to the unwanted kind of telemetry... `setup-` will be good.